editor/plugins/theme_editor_preview.cpp

```diff
@@ -217,7 +217,7 @@ void ThemeEditorPreview::_notification(int p_what) {
 			theme_cache.preview_picker_overlay_color = get_theme_color(SNAME("preview_picker_overlay_color"), SNAME("ThemeEditor"));
 			theme_cache.preview_picker_label = get_theme_stylebox(SNAME("preview_picker_label"), SNAME("ThemeEditor"));
 			theme_cache.preview_picker_font = get_theme_font(SNAME("status_source"), EditorStringName(EditorFonts));
-			theme_cache.font_size = get_theme_font_size(SNAME("font_size"), EditorStringName(EditorFonts));
+			theme_cache.font_size = get_theme_default_font_size();
```

Comment:
This was added in https://github.com/godotengine/godot/pull/54221, but there is no `font_size` in `EditorFonts`. I assume the intent here is to get some fallback value, hence my suggested change. cc @bruvzg

Reply:
Since the font size value is always used with `preview_picker_font` which is derived from `status_source`, maybe the `status_source_size` font size is the intended value to use here?

Reply:
I see. Well, in that case I think that my suggested change should be fine for now. Just because I use `status_source` font for the label in the picker mode, it doesn't mean I have to use its corresponding font size. The default one fits the design of the tool, so I'll leave it as is.

Thanks for feedback!